### ISSUE TITLE
New logo of Bitvavo and custom Verge link

### ISSUE THREAD

**Issue by noahpierau** [vergecurrency/vergecurrency.com#800]
Dear XVG team,

We are contacting you on behalf of Bitvavo, one of the biggest regulated EUR exchanges in Europe. On your website we saw this page https://vergecurrency.com/get-verge/ where you showcase several exchange options for XVG. 

Bitvavo is proud to be on that list! 

Recently we launched a new website with a redesigned logo. We encourage all listed projects to update our logo. Here is a .png version of the new logo you could use: https://bitvavo.com/press/blue/bitvavo-mark-square/bitvavo-mark-square-blue.png (other colors and file formats: https://bitvavo.com/en/press).

Apart from that, we prepared a custom asset page for you to link to: https://bitvavo.com/en/verge (for localized versions of the website just replace "en" with "de", "es", "fr", "it" or "nl", also reachable via the footer). If you see any inaccurate information on there, please do let us know and we will update it as soon as possible. 

TL;DR: we encourage you to use our [new logo](https://bitvavo.com/en/press) and link to https://bitvavo.com/en/verge

Kindly let us know if you have any questions.
